### DISCUSSION TITLE
Instances were predicted on 0 frames

### DISCUSSION THREAD

**Discussion by Mitchellb16** [talmolab/sleap#1125] [Help!]
I'm troubleshooting the message "Instances were predicted on 0 frames".

Some details that I am unclear on that may be causing issues: my videos are of a reaching task, many (~30%) of frames during which the subject/instance (arm) is not in view, I have added an instance on these frames and toggled all the labels to off. Is this correct?

I have labeled 40 total frames and trained using single animal pipeline type with .1 validation fraction and .5 input scaling. All other parameters have been left to default.

Here I've pasted the single instance model configuration and training pipeline

![image](https://user-images.githubusercontent.com/55215406/213286387-8c351e9f-344a-4787-a469-34c0df09c7ae.png)

![image](https://user-images.githubusercontent.com/55215406/213286482-d06396a3-10f0-44aa-9172-c78e570a1602.png)

**Comment by roomrys** [comment]
Hi @Mitchellb16,

So the negative examples (with all nodes marked as occluded) will not do anything at the moment as we remove them before feeding examples into the training pipeline. The size of the receptive field (depicted by the blue box overlaid on the image in the Training Pipeline) is very important in single animal and bottom-up models. The receptive field (RF) should be about the size of the animal you want to track. If the RF is too small, the network will have trouble finding the animal. 

Try increasing the RF by setting the max stride to 32. You can also try setting the filter rate to 1.5.

If you are able to provide a screenshot of your data (with the RF overlaid from w/in the Training Pipeline) that would help us diagnose further. (You can email this to [lmaree@salk.edu](mailto:lmaree@salk.edu) if you want to keep your data s bit more private.)

Let us know if this helps!

Thanks,
Liezl
